dev/TreeView/TreeView.cpp

```diff
@@ -70,6 +91,38 @@ winrt::IVector<winrt::TreeViewNode> TreeView::SelectedNodes()
     return m_pendingSelectedNodes.get();
 }
 
+void TreeView::SelectedItem(winrt::IInspectable const& item)
+{
+    if (auto listControl = ListControl())
+    {
+        auto node = listControl->IsContentMode() ? listControl->NodeFromItem(item) : safe_try_cast<winrt::TreeViewNode>(item);
+        listControl->SelectedItem(node);
+    }
+}
+
+winrt::IInspectable TreeView::SelectedItem()
+{
+    if (auto listControl = ListControl())
+    {
+        return listControl->SelectedItem();
+    }
+
+    return nullptr;
+}
+
+winrt::IVector<winrt::IInspectable> TreeView::SelectedItems()
+{
+    if (auto listControl = ListControl())
+    {
+        if (auto viewModel = listControl->ListViewModel())
+        {
+            return viewModel->GetSelectedItems();
+        }
+    }
+
+    return nullptr;
```

Comment:
would it make sense to return an empty list if there are no selected items, so that the consuming code does not have to do a null check ?

Reply:
yeah it's returning empty list in normal cases. This nullptr only happens when listControl or viewModel is not ready.